static/app/utils/analytics/onboardingAnalyticsEvents.tsx

```diff
@@ -33,6 +41,9 @@ export const onboardingEventMap: Record<keyof OnboardingEventParameters, string>
   'onboarding.explore_sentry_button_clicked': 'Onboarding: Explore Sentry Button Clicked',
   'onboarding.first_error_received': 'Onboarding: First Error Received',
   'onboarding.first_error_processed': 'Onboarding: First Error Processed',
+  'onboarding.js_loader_show_configuration':
+    'Onboarding: JS Loader Configuration Expanded',
```

Comment:
hm, shall we add the 'optional' word to be more explicit?

Reply:
I generally renamed them a bit to align better with other names!

Reply:
I see but the configuration we are talking is the "advanced" one and it would make things more clear imo

Reply:
What I meant is I changed it based on your feedback :D it is now `onboarding.js_loader_optional_configuration_shown`, is that better?

Reply:
oh I see....yes it is better...thanks!